### ISSUE TITLE
command line for training

### ISSUE THREAD

**Issue by a-maci** [Zehaos/MobileNet#9]
What is the exact command line you used to train (for the run where you got 66.51% top-1 accuracy)?

Thanks

**Comment by Zehaos** [comment]
Hi, 
I train it using Rmsprop for about 125 epoch on Imagenet2012, with a large batch size.
I think you can use a simple preprocessing with a small batch size(e.g. 64.) to shorten the training time.

```
python train_image_classifier.py \
  --train_dir=${TRAIN_DIR} \
  --dataset_name=imagenet \
  --dataset_split_name=train \
  --dataset_dir=${DATASET_DIR} \
  --model_name=mobilenet \
  --preprocessing_name=mobilenet \
  --width_multiplier=1.0 \
  --max_number_of_steps=2000000 \
  --batch_size=128 \
  --save_interval_secs=240 \
  --save_summaries_secs=240 \
  --log_every_n_steps=100 \
  --optimizer=rmsprop \
  --rmsprop_decay=0.9 \
  --opt_epsilon=1.0\
  --learning_rate=0.1 \
  --learning_rate_decay_factor=0.1 \
  --momentum=0.9 \
  --num_epochs_per_decay=30.0 \
  --weight_decay=0.0 \
  --num_clones=2
```
